app/util/mapIconUtils.js

```diff
@@ -180,6 +188,11 @@ export function drawRoundIcon(tile, geom, type, customScale, platformNumber) {
           geom.y / tile.ratio,
         );
       }
+
+      if (icon && imageSize) {
+        // eslint-disable-next-line no-use-before-define
```

Comment:
This seems like an odd error to silence.

Reply:
Should I rather move the function?

Reply:
Yes please. This seems like a dangerous one to ignore. We should fix the code instead.